units/systemd-binfmt.path

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Path activation for additional binary formats
+Documentation=man:systemd-binfmt.service(8) man:binfmt.d(5)
+Documentation=https://www.kernel.org/doc/Documentation/binfmt_misc.txt
+
+[Path]
+DirectoryNotEmpty=/lib/binfmt.d
+DirectoryNotEmpty=/usr/lib/binfmt.d
+DirectoryNotEmpty=/usr/local/lib/binfmt.d
```

Comment:
Does systemd-binfmt read these directories? If so, man page should be corrected. If not, I think the .service file should be updated.